app/cafe.py

```diff
@@ -0,0 +1,32 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    NotWearingMaskError,
+    OutdatedVaccineError
+)

```

Comment:
Add a blank line to separate different groups of imports according to PEP-8 in the future.